maison-de-l-orient-et-de-la-mediterannee.csl

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-16"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded">
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="sort-only" default-locale="fr-FR" page-range-format="expanded" style_validates="true">
```

Comment:
no, remove that again. Validation refers to https://github.com/citation-style-language/styles/wiki/Validation